### DISCUSSION TITLE
Feature: Hierarchy in checklist notes

### DISCUSSION THREAD

**Discussion by diegogurpegui** [standardnotes/forum#3326] [Suggestions]
When using a checklist note it would be useful to have the ability to define a hierarchy within the items. Google Keep has this possibility.
The idea is that each item can be places _inside_ another item, the same way you do it when creating bullet lists in any rich text editor. Doing it with the "Tab" key could be an option, or just draging.
Also, when an item is checked, all the children are automatically checked.
Again, we can take inspiration from Google Keep.

**Comment by FractalHabits** [comment]
Yes please! RemNote does this well. Also being able to collapse children would be great!
